activestorage/lib/active_storage/reflection.rb

```diff
@@ -4,12 +4,20 @@ module ActiveStorage
   module Reflection
     class HasAttachedReflection < ActiveRecord::Reflection::MacroReflection # :nodoc:
       def variant(name, transformations)
-        named_variants[name] = NamedVariant.new(transformations)
+        named_variants[name] = NamedRepresentation.new(transformations)
+      end
+
+      def preview(name, transformations)
```

Comment:
sorry I just realized that support for preview was never working. I need to do add more unit tests in the one_test.rb and many_test.rb  to make sure the support is complete @jonathanhefner 

Im switching to draft